frontend/src/routes/manage/Video/Shared.tsx

```diff
@@ -154,9 +154,13 @@ const ManageVideoNav: React.FC<ManageVideoNavProps> = ({ event, active }) => {
         </LinkWithIcon>
     ));
 
+    const videoLink = event.hostRealms.length === 1
+        ? `${event.hostRealms[0].path.replace(/^\/$/, "")}/v/${id}`
```

Comment:
Didn't you **just** make it so this `replace` isn't necessary anymore? 🤔

Reply:
Nop, it still is. Previously, the search API was not consistent with our main one and that was fixed. Now the search API would also require this replace. Every time we run into something like this we think "oh why don't we change the behavior for the root realm to get rid of this special handling", but then there are all the other places that need special handling if we fixed it. Both ways of doing that are possible and both require special handling in some cases.

Reply:
Ye ok makes sense.